src/omotes_sdk/internal/worker/params_dict.py

```diff
@@ -37,10 +37,12 @@ def convert_params_dict_to_struct(params_dict: ParamsDict) -> Struct:
     normalized_dict: Dict[str, PBStructCompatibleTypes] = {}
 
     for key, value in params_dict.items():
-        if isinstance(value, datetime):
+        if type(value) is datetime:
             normalized_dict[key] = value.timestamp()
-        elif isinstance(value, timedelta):
+        elif type(value) is timedelta:
             normalized_dict[key] = value.total_seconds()
+        elif type(value) is int:
```

Comment:
It turns out that `isinstance(True, int) == True`. As in bool values in Python are also integers. Never knew this xD